platform-includes/getting-started-config/javascript.astro.mdx

```diff
@@ -1,4 +1,11 @@
-Get started by adding your DSN to your Astro config file (`astro.config.mjs`):
+Configuration should happen as early as possible in your application's lifecycle.
+
+To set up the Sentry SDK, register the Sentry integration and initialize the SDK for client and server in the root directory of your project:
+
+- Client-Side: `sentry.client.config.ts` containing a `Sentry.init` call
+- Server-Side: `sentry.server.config.ts` containing a `Sentry.init` call
```

Comment:
Can we somehow co-locate the code blocks with this paragraph? I would also err towards making the `defineConfig` the very first thing in this guide.

Reply:
like not having separate headings for integration, client and server?

Reply:
Yeah, I don't have hard opinions on how we format it but it felt a bit weird that there is the build-time config in between this paragraph and the related headings.